public/docs/css/main.css

```diff
@@ -29,6 +29,9 @@ body {
   line-height: 1.6;
   color: var(--color-text);
   background-color: var(--color-base-primary);
+  transition-property: background-color, color;
```

Comment:
Users sometimes switch of animations for various perceptual conditions. We have a block elsewhere that uses the "prefers reduce motion" flag - it would be great to make sure we use it wherever we animate, so users who have issues with animation can avoid it.